renovate.json

```diff
@@ -12,7 +12,8 @@
   "osvVulnerabilityAlerts": true,
   "lockFileMaintenance": {
     "enabled": true,
-    "schedule": ["before 6am on wednesday"]
+    "schedule": ["before 6am on wednesday"],
+    "commitMessageTopic": "group {{groupName}}"
```

Comment:
I am not sure if `groupName` will work here since `lockFileMaintenance` is separate from `packageRules`
consider using `packageFile`? https://docs.renovatebot.com/templates/#other-available-fields

Reply:
It's so strange. Looking at the console:

![image](https://github.com/user-attachments/assets/b58f63e0-9119-4cbc-8887-816926fa20dd)

(which is also how they render in https://github.com/google/osv.dev/issues/714)

the two without titles correlate with
- https://github.com/google/osv.dev/pull/2743
- https://github.com/google/osv.dev/pull/2741

The ones that have something useful in the title look like it is maybe `parentDir`? The fact that it's already including something sometimes makes me think that further adjustment of the string isn't the right answer, it's failing to set a variable already...

Reply:
after taking a detailed look on the PRs generated this morning:
 - the two lockfile maintenance PRs without group name did not list any update explicitly, the update is `lockFileMaintenance`
 - other PRs with group name also listed package updates besides `lockFileMaintenance`

so my guess is for purely lock file maintenance PR, the group name is not added (which we actually want and this PR should help). For PRs with both lockfile maintenance and updates, the group names are added because this is supported to be a general update PR with grouping rules, but lockfile maintenance shouldn't be the PR main body.